build.gradle

```diff
@@ -38,7 +38,7 @@ subprojects {
 
   apply plugin: 'net.ltgt.errorprone'
   configurations.errorprone {
-    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.11'
+    resolutionStrategy.force 'com.google.errorprone:error_prone_core:2.0.5'
```

Comment:
I'll rebase anyway; but please rebase the branch (skipping this commit) to fix conflicts (in INSTALL.md, in other commits). Also, as you're at it, squash the last two commits into the appropriate previous commits to keep the history as clean as possible.